CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Features
```

Comment:
Need to rebase with main and fix the conflict here, cause we released 3.2.1 in the meantime